js/everything.js

```diff
@@ -137,6 +137,7 @@ var ZenPlayer = {
                 that.setupTitle();
                 that.setupVideoDescription(videoID);
                 that.setupPlyrToggle();
+                that.show();
```

Comment:
let's revert this change since our autoplay fix will eventually take care of it

Reply:
Done!